app/models/katello/input_template_renderer.rb

```diff
@@ -0,0 +1,7 @@
+class InputTemplateRenderer
+  include UnattendedHelper
+
+  def errata(id)
+    Katello::Erratum.find_by_errata_id(id).attributes.slice!('errata_id', 'created_at', 'updated_at')
```

Comment:
yeah, i'm fine with splicing leaving created_at and updated_at, i'd probably leave errata_id though.